controllers/managedocs_controller.go

```diff
@@ -1057,6 +1067,14 @@ func (r *ManagedOCSReconciler) reconcilePrometheus() error {
 			},
 			Key: alertRelabelConfigSecretKey,
 		}
+		if r.DeploymentType != convergedDeploymentType {
+			if remoteWriteSpec, err := r.getPrometheusRemoteWriteSpec(); remoteWriteSpec != nil {
+				r.Log.Info("Prometheus remote write spec fetched , will enable remote write")
+				r.prometheus.Spec.RemoteWrite = append(r.prometheus.Spec.RemoteWrite, *remoteWriteSpec)
+			} else {
+				r.Log.Info("Unable to fetch Prometheus Remote write spec, will disable remote write", "prom-remote-write-spec-error", err)
```

Comment:
This is an info message, lets structure it in that manner
```suggestion
				r.Log.Info("RHOBS remote write configuration is unavailable. disabling remote write", "prom-remote-write-spec-error", err)
```

Reply:
DOne